app/scripts/filters/resources.js

```diff
@@ -1328,4 +1328,14 @@ angular.module('openshiftConsole')
       var serviceClassDisplayName = _.get(serviceClasses, [serviceClassName, 'externalMetadata', 'displayName']);
       return serviceClassDisplayName || serviceClassName || instanceName;
     };
+  })
+  .filter('serviceInstanceStatus', function(isServiceInstanceReadyFilter) {
+    return function(instance) {
+      return isServiceInstanceReadyFilter(instance) ? 'Ready' : 'Pending';
```

Comment:
We'll need to check for the `Failed` condition as well as @jeff-phillips-18 added in #1961